x/blobstream/keeper/keeper.go

```diff
@@ -56,7 +56,7 @@ func (k Keeper) DeserializeValidatorIterator(vals []byte) stakingtypes.ValAddres
 	return validators
 }
 
-// StakingKeeper restricts the functionality of the bank keeper used in the qgb
+// StakingKeeper restricts the functionality of the bank keeper used in the Blobstream
```

Comment:
[question] This capitalization of `Blobstream` is not consistent with the previous capitalization of `qgb`, if the current edition is the correct one, then all good.

Reply:
good catch, when referring to the module, we should lowercase, I guess
```suggestion
// StakingKeeper restricts the functionality of the bank keeper used in the blobstream
```